base/cvd/cuttlefish/host/libs/web/cas/cas_flags.h

```diff
@@ -24,6 +24,24 @@
 
 namespace cuttlefish {
 
+// Wrapper for flag values that tracks whether a value was user-specified
+// (via command-line or config file) versus using a default.
+template <typename T>
+class FlagValue {
+ public:
+  FlagValue() : default_value_{} {}
```

Comment:
Don't provide a default constructor if it's not needed. Without it, if you add a new flag to `CasDownloaderFlags` but forget to initialize it in the constructor with a default value you'll get a compiler error; otherwise it will just use the default constructor.